forge-game/src/main/java/forge/game/card/CardCopyService.java

```diff
@@ -339,9 +346,15 @@ public Card getLKICopy(Map<Integer, Card> cachedMap) {
         newCopy.setIntensity(copyFrom.getIntensity(false));
         newCopy.setPerpetual(copyFrom);
 
-        newCopy.addRemembered(copyFrom.getRemembered());
-        newCopy.addImprintedCards(copyFrom.getImprintedCards());
-        newCopy.setChosenCards(copyFrom.getChosenCards());
+        Map<Card, GameCommand> extraEffects = Maps.newHashMap();
+        for (Card e : copyFrom.getExtraEffects()) { // should not need to run the game commands for LKI
```

Comment:
- I don't like all that extra for a playtest card
- not quite sure how this is supposed to work, but the concept of LKI effects seems a bit exotic rules wise
- switching Remembered etc. to extra LKI without good reason seems wrong, nothing says LKI isn't allowed to reference ingame objects

Reply:
I need it for other effects too, like the #6811 right now doesn't work because I need to create an effect for the spell you would cast
(And then it did flicker with the type)

Reply:
The problem with Remembered is this:
When Card A with internal Effect B makes a LKI copy of A, and A->B is linked, I need to copy B too. But the copy of B still points to original A, not the copy of A